analyzer/src/funTest/assets/projects/external/git-repo-expected-output.yml

```diff
@@ -377,7 +377,7 @@ analyzer:
         path: ""
       homepage_url: ""
       scopes: []
-    - id: "Unmanaged:manifest.xml:manifest:<REPLACE_REVISION>"
+    - id: "Unmanaged:manifest.xml:manifest:31588aa8f8555474e1c3c66a359ec99e4cd4b1fa"
```

Comment:
Is it intended that this gets expanded as part of this commit? Similar below.

Reply:
No it wasn't. Thanks for highlighting this! 

I've addressed it here: https://github.com/oss-review-toolkit/ort/pull/6429